apps/dev/src/filter-field/filter-field-demo.component.ts

```diff
@@ -152,6 +152,10 @@ export class FilterFieldDemo implements AfterViewInit, OnDestroy {
         this._firstTag = tags[0];
       }
     });
+
+    this.filterField.interactionStateChange.subscribe((isInteracted) => {
```

Comment:
Will be removed with rebase.

Reply:
We could leave that actually

Reply:
we can leave that in the devapp, you are right